### PR TITLE
yank

### PR DESCRIPTION
**Pull request by rayegun**
@vchuravy

### CODE REVIEW

File: P/PARMETIS_jll/Compat.toml [hunk 1]
```diff
@@ -9,6 +9,5 @@ julia = "1.6.0-1"
 
 ["4.0.5-4"]
 JLLWrappers = "1.4.0-1"
-METIS_jll = "5.1.2-5"
```
Comment (fredrikekre) [inline P/PARMETIS_jll/Compat.toml:12]: Why this change?

Comment (vchuravy) [reply P/PARMETIS_jll/Compat.toml:12]: This was spuriously introduced and the reason for the yank. Previous build IDs didn't have this dependency.

We should have done a version bump instead
